src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, onTabSelected, selectedTabId }) => {
+  const currentTab = tabs.find(tab => selectedTabId === tab.id) || tabs[0];
```

Comment:
You use the same find algorithm twice, this can be transferred to a separate helper function and reused in both components.

Reply:
Good point, fix it